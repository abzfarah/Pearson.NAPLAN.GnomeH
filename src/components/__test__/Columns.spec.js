// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react/lib/ReactTestRenderer';

import Columns from '../Columns';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('Columns', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Columns>
        <span>test1</span>
        <span>test2</span>
      </Columns>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
