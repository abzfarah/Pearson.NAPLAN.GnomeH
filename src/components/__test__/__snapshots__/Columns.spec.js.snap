exports[`Columns has correct default options 1`] = `
<div
  className="grommetux-columns grommetux-columns--justify-start grommetux-columns--responsive">
  <div
    className="grommetux-columns__column">
    <span>
      test1
    </span>
    <span>
      test2
    </span>
  </div>
</div>
`;
